lib/global.gd

```diff
@@ -155,6 +170,26 @@ DeclareGlobalFunction("MakeReadWriteGlobal");
 
 #############################################################################
 ##
+#F  MakeConstantGlobal( <name> )   . . . . make a global variable constant
```

Comment:
again the `. . . ` business...  should change the triple spaces to a single one and insert more `. ` -- the same in other places, too, of course

Reply:
I copied the triple spaces from the function directly above, and below, this one. This file has either triple spaces, or double spaces, never a single one.

Reply:
There are 25 places in the whole library using a triple space in that location, i.e. less than 1% -- I think they come from somebody not being careful and/or not caring.

Now, I am first to agree that it sucks to manually update these kinds of comments. But I still think there are only two sane choices: 
1. Abolish them, possibly incrementally (by just not adding them to new code)
2. Keep using them "as intended" -- and then triple spaces to me look quite clearly like a bug, which simply proliferates by copy&paste.

tl;dr: please either delete this line, or format it to match the rest of the library (and not just the rest of this comparatively short file).